imports/ui/QuestionSidebar.jsx

```diff
@@ -97,7 +103,13 @@ export class QuestionSidebar extends ControlledForm {
    * @param {Event} e
    */
   setType (e) {
-    this.setState({ questionType: parseInt(e.target.value) }, () => {
+    this.setState({ questionType: parseInt(e.target.value), courseId: this.props.courseId }, () => {
+      this.props.updateQuery(this.state)
+    })
+  }
+
+  setApproved () {
```

Comment:
In general, the use of the word Approved in this component is confusing as it implies that one is setting the approved property of a question. 
Maybe it should be something like showApproved.